src/lib/chat/sendbird-client.ts

```diff
@@ -86,6 +86,11 @@ export class SendbirdClient implements IChatClient {
     }
   }
 
+  async getUserPresence() {
```

Comment:
Not sure if having a stub/no-op is ideal here, but in the long run, maybe doesn't matter too much? Suggestions welcome!

Reply:
Updated.